using-openspace/scripting/index.md

```diff
@@ -0,0 +1,10 @@
+# Scripting
+
```

Comment:
This page should have some top level information, but maybe that's for a later stage?

Reply:
Yes, this PR does not add any hand written documentation